src/rantanplan/core.py

```diff
@@ -44,12 +44,12 @@
     ([aeiou][äëïöü])|
     #10: Explicit hiatus with umlaut vowels, second part
     ([äëïöü][a-z])|
-    #12: any char
+    #11: any char
     ([a-záéíóúñ])
 """, re.I | re.U | re.VERBOSE)  # re.VERBOSE is needed to be able to catch the regex group
 
 """
-Metrical Analysis
+rhythmical Analysis
```

Comment:
Why the change? :/ Also, please, capitalize.

Reply:
We decided to change the name, can't remember why. We also use "rhythmical_pattern"

Reply:
Some info on why would be handy.

Reply:
We changed the name of every "metric_whatever" to "rhytmmical_whatever" so to make it consistent. I don't understand what do you mean about info, info where?